src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/OpenDocumentGeneratorTest.cs

```diff
@@ -142,8 +141,7 @@ await projectManager.UpdateAsync(updater =>
             updater.DocumentAdded(_hostProject1.Key, _documents[0], _documents[0].CreateEmptyTextLoader());
 
             // Act
-            updater.ProjectWorkspaceStateChanged(_hostProject1.Key,
-                ProjectWorkspaceState.Create(LanguageVersion.CSharp8));
+            updater.ProjectWorkspaceStateChanged(_hostProject1.Key, ProjectWorkspaceState.Default);
```

Comment:
Does this actually do anything? I'm assuming that new projects are initialized with `ProjectWorkspaceState.Default`, so this no longer represents project change.

Reply:
Fair point. Updated this to change the configuration so the test is back to updating the C# version, as before